<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdatePluginProcessDialogue</class>
 <widget class="QDialog" name="UpdatePluginProcessDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="TxtDoing">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="TotalProgress">
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="StateListView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
